flipperdevices/flipperzero-firmware
# [FL-2654] Updater: retrying pre-boot SD card mount multiple times

# What's new

- [ FL-2654] Fixed updated being broken on certain SD cards
- Unified SD card mount sequence for pre-boot & main firmware
- Added BSP_SD_MaxMountRetryCount returning "10" to control # of retries

# Verification 

- Find an Sd card that fails to mount on first try
- Try updating to dev with OTA (should fail)
- Install this branch & try updating again (should work)

# Checklist (For Reviewer)
